CMakeLists.txt

```diff
@@ -11,6 +11,12 @@ option(NO_WEBSOCKET "Disable WebSocket support" OFF)
 option(NO_EXAMPLES "Disable examples" OFF)
 option(NO_TESTS "Disable tests build" OFF)
 option(WARNINGS_AS_ERRORS "Treat warnings as errors" OFF)
+option(CAPI_STDCALL "Set calling convention of C API callbacks stdcall" OFF)
+
+message("CAPI_STDCALL: ${CAPI_STDCALL}")
+if(CAPI_STDCALL)
+	add_definitions(-DCAPI_STDCALL)
```

Comment:
Since you would want that anything importing the library sets the definition too (otherwise it would crash) I think it would be more adapted to replace `add_definitions` with this after the declaration of targets:
```
target_compile_definitions(datachannel PUBLIC CAPI_STDCALL)
target_compile_definitions(datachannel-static PUBLIC CAPI_STDCALL)
```

Reply:
It definitely makes sense to have targets instead of making the definition global. I committed applying your suggestion.